includes/class-wc-payments.php

```diff
@@ -487,10 +495,13 @@ public static function init() {
 
 		self::maybe_register_platform_checkout_hooks();
 
-		// Payment Request and Apple Pay.
-		self::$payment_request_button_handler   = new WC_Payments_Payment_Request_Button_Handler( self::$account, self::get_gateway() );
+		self::$apple_pay_registration = new WC_Payments_Apple_Pay_Registration( self::$api_client, self::$account, self::get_gateway() );
+
+		self::$payment_request_button_handler = new WC_Payments_Payment_Request_Button_Handler( self::$account, self::get_gateway() );
```

Comment:
```suggestion
		$payment_request_button_handler = new WC_Payments_Payment_Request_Button_Handler( self::$account, self::get_gateway() );
```
Do `$payment_request_button_handler` and `$platform_checkout_button_handler` still need to be static variables? I can't see them used anywhere else in this class and it looks like the `WC_Payments_Express_Checkout_Button_Display_Handler` now handles functionality once encapsulated by these other two classes.

Reply:
Good point! There's no reason for `$payment_request_button_handler` and `$platform_checkout_button_handler` to remain static variables since they're just being passed into the constructor of `WC_Payments_Express_Checkout_Button_Display_Handler` and not requiring any direct access via the class.